docs/rst/notes/notes.rst

```diff
@@ -78,3 +78,11 @@ This release includes the following **Examples**:
 This release includes the following **Documentation features**:
 
 * This same documentation.
+
```

Comment:
Remove this as the forthcoming version section is already included in parent file.